src/components/internal-event/internal-event.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const InternalEvent = ({ event }) => {
```

Comment:
I would prefer to call this component just event-box. What do you think? 

If it is internal or external it depends on the source data

Reply:
Done